webui/react/src/hooks/useModal/Project/useModalProjectDelete.tsx

```diff
@@ -39,8 +39,10 @@ const useModalProjectDelete = ({ onClose, project }: Props): ModalHooks => {
 
   const handleOk = useCallback(async () => {
     try {
-      await deleteProject({ id: project.id });
-      routeToReactUrl(paths.workspaceDetails(project.workspaceId));
+      const response = await deleteProject({ id: project.id });
+      if (response.completed) {
```

Comment:
question: is it correct to do nothing if `response.completed` is false?

Reply:
I'm open to `routeToReactUrl(paths.workspaceDetails(project.workspaceId));` in both cases. As a tester it was helpful to stay on the page and see that the experiments were getting deleted.

Reply:
My main concern is what happens once all of the experiments are deleted? Does the page just error out? Your call, ulitmately.

Reply:
OK, I will remove the change to URL behavior on both modals. The only changes to the modals will be saying that it deletes all experiments. The delete option is appearing in WebUI only for workspaces/projects without experiments, so it will just be the wording changing.